src/minimega/main.go

```diff
@@ -50,6 +50,7 @@ var (
 	f_cli        = flag.Bool("cli", false, "validate and print the minimega cli, in JSON, to stdout and exit")
 	f_panic      = flag.Bool("panic", false, "panic on quit, producing stack traces for debugging")
 	f_cgroup     = flag.String("cgroup", "/sys/fs/cgroup", "path to cgroup mount")
+	f_future     = flag.Bool("test-the-future", false, "disable features that will be removed in future releases, such as targeting VMs by ID")
```

Comment:
It does log a warning if they target VMs by ID and don't specify the flag.

The flag disallows targeting VMs by ID and does not log a warning, so it achieves the same behavior as the future release in which targeting VMs by ID is disabled.

Or did you mean there should be a flag purely to disable the warnings, and still allow targeting VMs by ID?